server/session_ws.go

```diff
@@ -117,6 +117,7 @@ func (s *sessionWS) Consume(processRequest func(logger *zap.Logger, session Sess
 	s.conn.SetReadDeadline(time.Now().Add(time.Duration(s.config.GetSocket().PongWaitMs) * time.Millisecond))
 	s.conn.SetPongHandler(func(string) error {
 		s.conn.SetReadDeadline(time.Now().Add(time.Duration(s.config.GetSocket().PongWaitMs) * time.Millisecond))
+		s.logger.Debug("Received WebSocket PONG")
```

Comment:
Do we need this still? I vote we remove it as it effectively makes the debug logs very very verbose.